Breakpoint 1 at 0x14d5: file Assignment_3.cpp, line 43.
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGINT, Interrupt.
0x00007ffff7c13a61 in __GI___libc_read (fd=0, buf=0x55555556e6c0, nbytes=1024) at ../sysdeps/unix/sysv/linux/read.c:26
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 > input.txt
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
0x0000555555555b18 in std::vector<int, std::allocator<int> >::size (this=0x0) at /usr/include/c++/13/bits/stl_vector.h:993
Quit
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[Inferior 1 (process 7384) exited normally]
Breakpoint 1 at 0x5555555554d5: file Assignment_3.cpp, line 43.
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
0x0000555555555b18 in std::vector<int, std::allocator<int> >::size (this=0x0) at /usr/include/c++/13/bits/stl_vector.h:993
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 > input.txt
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGINT, Interrupt.
0x00007ffff7c13a61 in __GI___libc_read (fd=0, buf=0x55555556f2c0, nbytes=1024) at ../sysdeps/unix/sysv/linux/read.c:26
Breakpoint 1 at 0x14d5: file Assignment_3.cpp, line 43.
Starting program: /home/rohithk/mcw/FTTU-interns/rohith_krishnan/gdb_assignment_gets2025/Assignment_3/Assignment_3 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Breakpoint 1, maxAreaOfIsland (grid=std::vector of length 8, capacity 8 = {...}) at Assignment_3.cpp:43
No symbol "max_answer" in current context.
Hardware watchpoint 2: max_number

Hardware watchpoint 2: max_number

Old value = 0
New value = 1
maxAreaOfIsland (grid=std::vector of length 8, capacity 8 = {...}) at Assignment_3.cpp:44
Continuing.

Hardware watchpoint 2: max_number

Old value = 1
New value = 4
maxAreaOfIsland (grid=std::vector of length 8, capacity 8 = {...}) at Assignment_3.cpp:44
Continuing.

Hardware watchpoint 2: max_number

Old value = 4
New value = 5
maxAreaOfIsland (grid=std::vector of length 8, capacity 8 = {...}) at Assignment_3.cpp:44
Continuing.

Hardware watchpoint 2: max_number

Old value = 5
New value = 6
maxAreaOfIsland (grid=std::vector of length 8, capacity 8 = {...}) at Assignment_3.cpp:44
Continuing.

Watchpoint 2 deleted because the program has left the block in
which its expression is valid.
0x00005555555558fa in main () at Assignment_3.cpp:88
Continuing.
[Inferior 1 (process 8293) exited normally]
